webui/react/src/utils/experiment.ts

```diff
@@ -23,3 +24,26 @@ export const isSingleTrialExperiment = (experiment: ExperimentBase): boolean =>
   return experiment?.config.searcher.name === ExperimentSearcherName.Single
         || experiment?.config.searcher.max_trials === 1;
 };
+
+export const trialHParamsToExperimentHParams = (
```

Comment:
suggestion: I'd name the part that's unflattening the dictionary something more generic and separate from trialparams and give it a comment. maybe make `.` a paramter to it.

Reply:
I can't remember does the flattening of the hparams happen anywhere in our code? if it did using this we could add a set of experiment configs with nested hyperparams to our fixtures and make sure that`forEach config c => unflatten(flatten(c)) === c`

Reply:
good thing you mentioned this, totally forgot about `flattenObject` in `utils/data.ts`!

Reply:
looks good : )